modules/InteractiveMusic/SCsub

```diff
@@ -0,0 +1,5 @@
+# SCsub
+Import('env')
+
+src_list = ["audio_stream_playlist.cpp", "audio_stream_transitioner.cpp", "register_types.cpp"]
+env.add_source_files(env.modules_sources, src_list)
```

Comment:
It's better to add sources to module env by cloning `env_modules` as not to interfere with the rest of the build, also you can add sources using a wildcard pattern instead since they are all within module's root:
```python
Import('env')
Import('env_modules')

env_music = env_modules.Clone()
env_music.add_source_files(env.modules_sources, "*.cpp")
```